### ISSUE TITLE
Feature request: ability to run autoimport on a directory

### ISSUE THREAD

**Issue by Jasha10** [lyz-code/autoimport#141]
## Description
It would be nice if calling `autoimport <directory>` (on the command line) would process the python files under that directory.

## Possible Solution
I find myself typing this at the command line sometimes: 
```$ for f in `ls *.py`; do autoimport $f; done```

## Additional context
This would be consistent with other tools:
```text
mypy <directory>
flake8 <directory>
black <directory>
isort <directory>
# etc...
```

**Comment by lyz-code** [comment]
Hi @Jasha10, thanks for taking the time to open the issue, the behaviour described makes all the sense. Sadly, I won't be able to look into it this month as I'm a little bit overloaded.

If you can't wait and have coding skills and some time, feel free to make a pull request.

**Comment by lyz-code** [comment]
`autoimport` accepts many files though, so if you're in a directory, you can do `autoimport *.py`. But it won't recursively search in the subdirectories, which would be ideal

**Comment by superDross** [comment]
The above PR should facilitate the requested behavior.

I am struggling with getting this to work with `mypy`. Specifically the below error:
```
src/autoimport/entrypoints/cli.py:38: error: Incompatible return value type (got "Tuple[Any, ...]", expected "Tuple[Any]")  [return-value]
Found 1 error in 1 file (checked 6 source files)
```

I have been trying to resolve this for over an hour and cannot proceed with altering the types to get it to work. Any help would be appreciated.

The tests and linters are passing (at least locally) so this is the last issue before it will completely pass the pipeline.

**Comment by lyz-code** [comment]
Sorry @superDross I didn't see this comment before answering to the PR, as I see from [this stack overflow question](https://stackoverflow.com/questions/772124/what-does-the-ellipsis-object-do) the ellipsis is used in this case for :

> Arbitrary-length homogeneous tuples can be expressed using one type and ellipsis, for example Tuple[int, ...]

What I understand from that is that as we don't know the number of arguments of the tuple, that is the correct function signature. you'll probably need to change it downstream too

**Comment by superDross** [comment]
I have updated the PR and it is now passing the pipeline.

**Comment by lyz-code** [comment]
Thank you to @superDross the feature is available under `1.0.0`!! 

I've decided to go with `1.0.0` because it looks like enough people is using it and the basic features are ready.

Thanks @Jasha10 @dbatten5 @JaapJoris @superDross @fsouza @HoverHell @jugmac00 @uuwaan for making it possible
